2_Controllers/test_controller.js

```diff
@@ -1,12 +1,19 @@
-const { fetchTopics, fetchEndpoints } = require("../1_Models/test_model");
-const { request, response } = require("../app");
+const {
+  fetchTopics,
+  fetchEndpoints,
+  fetchArticlesById,
+} = require("../1_Models/test_model");
+// const { request, response } = require("../app");
```

Comment:
Please clean up these require statements, we don't need to bring in anything from app.js or any of the data from our db folder.